rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -502,8 +502,8 @@ namespace rsx
 #endif
 			u64 start_time = get_system_time();
 
-			const u64 period_time = 1000000 / g_cfg.video.vblank_rate;
-			const u64 wait_sleep = period_time - u64{period_time >= host_min_quantum} * host_min_quantum;
+			u64 period_time = 1000000 / g_cfg.video.vblank_rate;
```

Comment:
If you need to update this every cycle then move the declaration inside the loop [here](https://github.com/RPCS3/rpcs3/pull/7331/files#diff-646d2875fee53ff3f6de5b4a68c7adc5R511) and avoid duplicate code.